src/graphics/renderer.cpp

```diff
@@ -1,12 +1,21 @@
 #include "graphics/renderer.h"
-
```

Comment:
Why? I think it makes sense to separate internal dependencies from standard library dependencies using additional newline.

Reply:
done, it was missclick